spec/core/AsyncExpectationSpec.js

```diff
@@ -235,6 +225,583 @@ describe('AsyncExpectation', function() {
     });
   });
 
+  describe('async matchers', function() {
```

Comment:
These specs look good except that they fail in environments that don't support promises, notably IE. You can fix that by adding a call to `jasmine.getEnv().requirePromises()` either to each spec or to a `beforeEach`.